runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -3260,175 +3289,196 @@ TR::CompilationInfoPerThreadRemote::processEntry(TR_MethodToBeCompiled &entry, J
          J9::IlGeneratorMethodDetailsType, std::vector<TR_OpaqueClassBlock*>,
          JITaaSHelpers::ClassInfoTuple, std::string, std::string, uint32_t>();
 
-      uint64_t clientId         = std::get<0>(req);
+      clientId                  = std::get<0>(req);
       uint32_t romMethodOffset  = std::get<1>(req);
       J9Method *ramMethod       = std::get<2>(req);
-      J9Class *clazz            = std::get<3>(req);
-      TR_Hotness optLevel       = std::get<4>(req);
-      std::string detailsStr    = std::get<5>(req);
-      auto detailsType          = std::get<6>(req);
-      auto &unloadedClasses     = std::get<7>(req);
-      auto &classInfoTuple      = std::get<8>(req);
-      std::string clientOptStr  = std::get<9>(req);
-      std::string recompInfoStr = std::get<10>(req);
-      uint32_t seqNo            = std::get<11>(req); // sequence number at the client
-
-      //if (seqNo == 100)
-      //   throw JITaaS::StreamFailure(); // stress testing
-
-      stream->setClientId(clientId);
-      setSeqNo(seqNo); // memorize the sequence number of this request
-
-      bool sessionDataWasEmpty = false;
-      ClientSessionData *clientSession = NULL;
-      {
-      // Get a pointer to this client's session data
-      // Obtain monitor RAII style because creating a new hastable entry may throw bad_alloc
-      OMR::CriticalSection compilationMonitorLock(compInfo->getCompilationMonitor());
-      compInfo->getClientSessionHT()->purgeOldDataIfNeeded(); // Try to purge old data
-      if (!(clientSession = compInfo->getClientSessionHT()->findOrCreateClientSession(clientId, seqNo, &sessionDataWasEmpty)))
-         throw std::bad_alloc();
-
-      setClientData(clientSession); // Cache the session data into CompilationInfoPerThreadRemote object
-      if (TR::Options::getVerboseOption(TR_VerboseJITaaS))
-         TR_VerboseLog::writeLineLocked(TR_Vlog_JITaaS, "compThreadID=%d %s clientSessionData=%p for clientUID=%llu seqNo=%u",
-            getCompThreadId(), sessionDataWasEmpty ? "created" : "found", clientSession, (unsigned long long)clientId, seqNo);
-      } // end critical section
-
-     
-      // We must process unloaded classes lists in the same order they were generated at the client
-      // Use a sequencing scheme to re-order compilation requests
-      //
-      clientSession->getSequencingMonitor()->enter();
-      clientSession->updateMaxReceivedSeqNo(seqNo);
-      if (seqNo > clientSession->getExpectedSeqNo()) // out of order messages
+      if (!ramMethod)
```

Comment:
Just for aesthetics could we leave the string of std::get<> uninterrupted by this `if` statement?